stdlib/range.gr

```diff
@@ -1,8 +1,34 @@
+/**
+  @module Range: Utilities for working with ranges. A range represents an interval, or a set of values with a beginning and an end.
+  @example import Range from "range"
```

Comment:
I think this will even be compatible with multiline examples in the future! https://jsdoc.app/tags-example.html

Side note that's not relevant here... How do you escape within jsdoc to show an example with a decorator (thinking about Grain attributes) 🤔

Reply:
@ospencer multi-line anything is pretty tricky given my current regexp implementation. We probably won't want to look at a full-on lexer if we want to handle multi-line gracefully.

As for decorators, I only parse attributes at the very start of a line. I think multi-line examples should have to be indented (though, I also trim those lines...)